pkg/client/test_data/add_with_git_commit/test_pkg/kcl.mod.expect

```diff
@@ -4,4 +4,4 @@ edition = "0.0.1"
 version = "0.0.1"
 
 [dependencies]
-flask-demo-kcl-manifests = { git = "https://github.com/kcl-lang/flask-demo-kcl-manifests.git", commit = "ade147b" }
+flask-demo-kcl-manifests = { git = "https://github.com/kcl-lang/flask-demo-kcl-manifests.git", commit = "ade147b", version = "0.0.1" }
```

Comment:
Add more test cases for

1. Add dependencies from git repo by git commit, branch and tag, to test whether the `version` is updated to kcl.mod.

2. Add more test case for if the git dependency with `version` field in kcl.mod, `update` or `run` should check whether  version exists in the current commit, branch or tag.

Reply:
@zong-zhe I have added the the tests, Could you please review now!!!!